README.md

```diff
@@ -1,50 +1,70 @@
-# Prism
```

Comment:
I think the main title should still stay

Reply:
<img width="300" alt="Screenshot 2020-12-22 at 12 41 35 PM" src="https://user-images.githubusercontent.com/17899454/102862848-8b998300-4453-11eb-97e6-c0b453bd39c1.png">

It looks like this in Stoplight which is weird.

The change makes it look like this:
<img width="305" alt="Screenshot 2020-12-22 at 12 45 38 PM" src="https://user-images.githubusercontent.com/17899454/102862891-9ce28f80-4453-11eb-859e-c8fb365419cc.png">

Reply:
Well ok, but then I'd argue the bug is in the platform and should be fixed.

Reply:
Wouldn't say its a bug considering most tools would pick up the first heading.

If you think this is in a good spot, can we merge these changes in and make potential improvements later in another PR?